packages/pharos/src/pages/reports/react/ReportExample.tsx

```diff
@@ -0,0 +1,107 @@
+import { useEffect, type FC, useState } from 'react';
+import { Sidenav } from './Sidenav';
+import { CreateReportModal } from './CreateReportModal';
+import { ReportsTable } from './ReportsTable';
+import { historyTable, scheduledTable } from '../mocks';
+import logo from '@config/assets/images/reports/jstor-horizontal.svg';
+
+import {
+  PharosHeading,
+  PharosLink,
+  PharosButton,
+  PharosTabs,
+  PharosTab,
+  PharosTabPanel,
+  PharosToaster,
+  PharosLayout,
+} from '../../../react-components';
+
+export const ReportExample: FC = () => {
+  const mobileBreakpoint = 1055;
+  const [isMobile, setIsMobile] = useState<boolean>(window.innerWidth < mobileBreakpoint);
+  const [isSidenavDisplayed, setIsSidenavDisplayed] = useState<boolean>(!isMobile);
+
+  useEffect(() => {
+    const handleResize = () => {
+      const windowWidth = window.innerWidth;
+      setIsSidenavDisplayed(windowWidth >= mobileBreakpoint);
+      setIsMobile(windowWidth < mobileBreakpoint);
+    };
+
+    window.addEventListener('resize', handleResize);
+
+    return () => {
+      window.removeEventListener('resize', handleResize);
```

Comment:
Did a `ResizeObserver` approach not work well here?

Reply:
Oh, I would imagine it would, I was having trouble getting it working as intended and swapped it out to see if that was the issue, but it ended up being something else entirely and I just never swapped it back.

Reply:
Updated to use ResizeObserver again